app/services/hyrax/works/virus_checker_service.rb

```diff
@@ -0,0 +1,52 @@
+module Hyrax::Works
```

Comment:
could/should we do away with `Works` in these inlined files? just `Hyrax::VirusCheckerService`, `Hyrax::VirusScanner`, and `Hyrax::VirusCheck` seems better to me.

Reply:
It's done. Thanks.